.github/workflows/jira_issue.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           summary: '${{ github.event.issue.title }}'
           description: |
             ${{ github.event.issue.body }} \\ \\ _Created from GitHub Action_ for ${{ github.event.issue.html_url }}
-          fields: '{ "customfield_11401": {"id": "14583"}, "assignee": {"id": "61027a219798100070592b20"} }'
+          fields: '{ "customfield_11401": {"id": "14723"}, "assignee": {"id": "61027a219798100070592b20"} }'
```

Comment:
1. Can we change also entry in repo metadata?
2. Can we also set labels?

Reply:
I have created an issue to improve the integration: SNOW-1156105